app/activity/[id]/components/activity-card.tsx

```diff
@@ -0,0 +1,82 @@
+'use client'
+
+import { Heading, Box, Flex, Spacer, Text, IconButton } from '@chakra-ui/react'
+import { FiClock, FiMapPin, FiLink2, FiEdit3, FiTrash2 } from 'react-icons/fi'
+import { Link } from '@/components/link'
+import { customColors } from '@/theme/color'
+import { ActivityCollectionQuery } from '@generated/api'
+
+type ActivityInfoProps = {
+  activityData: ActivityCollectionQuery | undefined
+}
+
+export const ActivityCard: React.FC<ActivityInfoProps> = ({ activityData }) => {
+  if (!activityData || !activityData.activityCollection) {
+    return null
+  }
+
+  const data = activityData?.activityCollection?.edges[0]?.node
+  const timeFrom = data.time_from?.split('T')[0]
+  const timeTo = data.time_to?.split('T')[0]
+
```

Comment:
Managing date time from server response is supposed to be managed in `libs/utils/date.ts`, cuz this format doesn't come from JS API but PostgreSQL itself/supabase GraphQL, so I'd say put them internal logic would be better.

Reply:
okay! Under data.ts, I added 'formatDateToDayMonthYearTime' to format that has time (HH:mm) 
https://github.com/tabi-memo/frontend/pull/16/files#:~:text=export%20const%20formatDateToDayMonthYearTime,%7D